ui/js/perf.js

```diff
@@ -88,7 +88,7 @@ treeherder.factory('PhSeries', ['$http', 'thServiceDomain', 'ThOptionCollectionM
                                      return response.data;
                                  } else {
                                      return $q.reject("No data been found for job id " +
-                                                      jobId + " in project " + projectName);
+                                                      jobId + " in project " + projectName); //ShrutiJ: jobId throws no-undef
```

Comment:
This is a real bug, someone (probably me) must have changed the logic without handling the error case. I would change it to just $q.reject("No series data found")